recipe/meta.yaml

```
@@ -254,14 +256,14 @@ outputs:
         - cffi
         - cloudpickle
         - cython
-        - fastparquet
+        - fastparquet   # [py<314]
```

https://github.com/conda-forge/fastparquet-feedstock/pull/78 was merged, so can probably remove this selector?

```suggestion
        - fastparquet
```